lec_12_HW/task_1.ts

```diff
@@ -0,0 +1,75 @@
+// 1. Создайте дженерик функцию getFirstElement, которая принимает массив элементов типа T, и возвращает первый элемент (типа T).
+
+function getFirstElement<T>(array: T[]) {
+    return array[1];
```

Comment:
`JavaScript arrays are zero-indexed: the first element of an array is at index 0, the second is at index 1, and so on — and the last element is at the value of the array's length property minus 1.`